Python can handle our exceptions